queries/dune_v1/eth_spent.sql

```diff
@@ -0,0 +1,10 @@
+-- V1: https://dune.com/queries/1320169
+select concat('0x', encode(solver_address, 'hex')) as receiver,
+       sum(gas_price_gwei * gas_used) * 10 ^ 9     as amount
```

Comment:
FYI: You're missing the `num_transactions` column here compared to the query in the link above. Not sure if you need it or not.

Reply:
We don't actually need it, but I could add it back in. This was just an additional column used to help debug the query at one point. Probably wouldn't hurt to keep but the payouts script doesn't actually use it. In essence of the suspect API payment scheme, its probably better if we only fetch columns that are actually needed by the program (save on some returned bytes).